packages/docs/src/netlify/users.ts

```diff
@@ -130,16 +119,16 @@ async function getData({ sortBy, sortDesc, page, itemsPerPage }: DataOptions): P
 }
 
 const handler: Handler = async (event: HandlerEvent) => {
-	const options = event.queryStringParameters as unknown as DataOptions;
+	const options = getDataOptionsFromQueryString(event.queryStringParameters as EventQueryStringParameters);
 	const datas = await getData(options);
 
 	return {
 		headers: {
 			'Access-Control-Allow-Origin': process.env.API_HEADERS_ACCESS_CONTROL_ALLOW_ORIGIN as string
 		},
 		statusCode: 200,
-		body: JSON.stringify(datas),
+		body: JSON.stringify(datas)
 	};
-};
+}
```

Comment:
```suggestion
};
```
Ici il faut bien un point-virgule comme c'est une constante